webserver/templates/datasets/evaluate.html

```diff
@@ -21,6 +21,10 @@ <h2 class="page-title">Evaluate dataset "{{ dataset['name'] }}"</h2>
         <div class="col-sm-4">{{ form.filter_type(class="form-control", required="required") }}</div>
       </div>
       <div class="form-group">
+        <label class="col-sm-2 control-label">{{ form.evaluation_location.label.text }}</label>
+        <div class="col-sm-4">{{ form.evaluation_location(class="form-control", required="required") }}</div>
+      </div>
+      <div class="form-group">
```

Comment:
I think a help page describing in detail how to use this tool would be good. And giving links for the page on the evaluate page would be a good idea. Maybe we could give it a tag or something which says "Make this evaluation your contribution to AB by evaluating on your own machine", and then the user could click on that and get to the page where he/she can read the instructions.